src/Main.elm

```diff
@@ -71,6 +78,15 @@ update msg model =
                     , Browser.Navigation.load href
                     )
 
+        ToggleLanguage ->
+            ( if model.language == English then
+                Model Welsh
```

Comment:
The idiomatic way to write a change in record property so it is clear what is changing is 

`{model | language = Welsh}`